react/features/base/dialog/components/Dialog.web.js

```diff
@@ -128,15 +250,15 @@ class Dialog extends AbstractDialog {
      * @private
      */
     _renderOKButton() {
-        if (this.props.submitDisabled) {
+        if (this.props.okDisabled) {
```

Comment:
Are you sure this is not a regression? 

I noticed we are passing `okDisabled` to the button component. So maybe the idea before was:
if `submitDisabled` is set to true - don't show OK button
if `okDisabled` is set to true - show different style or something. 

I don't know if this was working like that, tho. 

Also if we need to check only `okDisabled` and not render the button why we are later passing it to the button component?